MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include README.rst
```

Comment:
It's not redundant but you're right to point it out. I added the README.md file to the manifest but I should have taken the ".rst" file off. I'm assuming that the project switch at some point.